sub_cal/sub_cal/src/main/java/validation/InputValidation.java

```diff
@@ -0,0 +1,24 @@
+package validation;
+
+import option.Option;
+
+import java.util.Optional;
+
+public class InputValidation {
+    public static boolean isValidMenuNumber(Optional<Option> userOption){
+        if(userOption.isEmpty()) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEmptyInputExpression(String expression){
+        if(expression.isEmpty()){
+            return true;
+        }
+        return false;
```

Comment:
[nit]
```suggestion
        if(expression.isBlank()){
            return true;
        }
        return false;
```

empty는 공백 값(white space)가 들어있는 문자열을 체크하지 못해서 `isBlank`를 고려해보는 것도 좋을 것 같아요!

Reply:
isEmpty만 알고있었는데 isBlank도 알아보겠습니다 !